pkgs/development/python-modules/spyder/default.nix

```diff
@@ -99,6 +92,7 @@ buildPythonPackage rec {
     textdistance
     three-merge
     watchdog
+    yapf
```

Comment:
Is this required?

Reply:
Probably not, but `lilv` is currently broken on staging, so I will have to wait to rebuild.

Reply:
What's `lilv`?
It seems that `yapf` is already included in `python-lsp-server.optional-dependencies.all`.